shaders/src/getters.fs

```diff
@@ -61,6 +61,25 @@ float getNdotV() {
     return shading_NoV;
 }
 
+/**
+ * @public-api
+ *
+ * In Vulkan and Metal, texture coords are Y-down. In OpenGL, texture coords are Y-up.
+ *
+ * When sampling from non-render targets (i.e. uploaded textures) this distinction does not matter
+ * because OpenGL has a second piece of backwardness, which is that the first row of texels
+ * in glTexImage2D is interpreted as the bottom row.
+ *
+ * To protect users from these differences, we recommend that materials in the SURFACE domain
+ * leverage this wrapper function when sampling from offscreen render targets.
+ */
+vec4 sampleRenderTarget2D(sampler2D samp, vec2 uv) {
```

Comment:
Maybe make note that these are normalized uv coordinates? In the future we might want to add a `texelFetchRenderTarget` method too that uses texel coordinates.

Reply:
Done.